src/utilities/bcl/templates/ReportingMeasure/tests/reporting_measure_test.rb

```diff
@@ -87,7 +71,19 @@ def setup_test(test_name, idf_output_requests, model_in_path = model_in_path_def
       end
     end
 
-    setup_test_2(test_name, epw_path)
+    osw_path = File.join(run_dir(test_name), 'in.osw')
+    osw_path = File.absolute_path(osw_path)
+
+    workflow = OpenStudio::WorkflowJSON.new
+    workflow.setSeedFile(File.absolute_path(model_out_path(test_name)))
+    workflow.setWeatherFile(File.absolute_path(epw_path))
+    workflow.saveAs(osw_path)
```

Comment:
We're creating OpenStudio::Workflow twice: in the body of the test, then in setup_test. Not that it matters that much though, just pointing it out.